accessibility/accessibility-checklist.md

```diff
@@ -39,6 +39,30 @@ Resources:
 * [Tool: HTML5 accessibility feature support for sections](http://www.html5accessibility.com/#sections)
 
 
+### Provide a Skip Navigation option
+
+Not all users are able to navigate by landmarks. For those users the multiple links typically found at the head of a document all have to be read via assistive technology, or laboriously tabbed through using a keyboard.
+
+To help them out you can add a "Skip Navigation" link to the top of each page which, when clicked, will take them to the main section of the page.
+
+We do this:
+
+```
+<body>
+<p><a href="#maincontent">Skip to main content</a></p>
+...
+<main id="maincontent">
+<h1>Heading</h1>
+<p>This is the first paragraph</p>
+```
+
+You should supply CSS that hides the Skip Navigation _for sighted users only_, and makes it highly visible when it receives focus via tabbing.
```

Comment:
Might be worth adding a bit here about how you can also use skip links to bypass other annoying things here (there's a bit in the WebAIM article about that, under the "multiple skip links" section)

Reply:
I think that potentially confuses things in what is a very brief section. That WebAIM section itself becomes incredibly nuanced, as they suggest _not_ adding more skip links, and instead using better document structure and in page navigation.